package.json

```diff
@@ -23,10 +23,12 @@
     "url": "https://github.com/wbuchwalter/tslint-loader/issues"
   },
   "homepage": "https://github.com/wbuchwalter/tslint-loader",
+  "peerDependencies": {
+    "tslint": "^3.0.0"
+  },
   "dependencies": {
     "loader-utils": "^0.2.7",
     "strip-json-comments": "^1.0.2",
-    "tslint": "^2.5.0-beta",
     "typescript": "^1.6.0-beta"
```

Comment:
Now that I think about it, I don't see a reason for TypeScript to be a (direct) dependency at all. After all, TSLint already asks for it. :neutral_face: